dev/com.ibm.ws.microprofile.openapi_fat/fat/src/com/ibm/ws/microprofile/openapi/validation/fat/OpenAPIValidationTestFive.java

```diff
@@ -0,0 +1,101 @@
+package com.ibm.ws.microprofile.openapi.validation.fat;
+
+import static org.junit.Assert.*;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.HttpUtils;
+
+/**
+ * A class to test the Tag, Discriminator, Schema and Extension validators.
+ * Scenarios include
+ * Tag: the name field is missing
+ * Discriminator: propertyName field is missing
+ * Schema: inappropriate fields for certain Schema types (min/max items or uniqueOnly fields on String type, min/max Length on array types)
+ *         invalid values for certain fields such as negative values for length
+ *         conflicting fields such as the readOnly and writeOnly fields
+ * Extension: name not starting wiht "x-"
+ *
+ */
+@RunWith(FATRunner.class)
+public class OpenAPIValidationTestFive {
+
+    @Server("validationServerFive")
+    public static LibertyServer server;
+    private static final String OPENAPI_VALIDATION_YAML = "Validation";
+
+    private static final int TIMEOUT = 10000; //in milliseconds (10 seconds)
+
+    @BeforeClass
+    public static void setUpTest() throws Exception {
+        HttpUtils.trustAllCertificates();
+
+        server.startServer("OpenAPIValidationTestFive.log", true);
+
+        server.validateAppLoaded(OPENAPI_VALIDATION_YAML);
+
+        assertNotNull("Web application is not available at /Validation/",
+                      server.waitForStringInLog("CWWKT0016I.*/Validation/")); //wait for endpoint to become available
+        assertNotNull("CWWKF0011I.* not recieved on relationServer",
+                      server.waitForStringInLog("CWWKF0011I.*")); // wait for server is ready to run a smarter planet message
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        server.stopServer("CWWKO1603E", "CWWKO1650E");
+    }
+
+    @Test
+    public void testStartup() throws Exception {
+
+        assertNotNull("The validation server did not start", server.waitForStringInLog("CWWKE0001I:.*"));
```

Comment:
Please add this line into `@BeforeClass`. There is no guarantee this test would be run first

Reply:
Please change it across the PR